package.json

```diff
@@ -11,6 +11,7 @@
     "@patternfly/react-table": "5.0.1",
     "@patternfly/react-tokens": "5.0.1",
     "deep-equal": "2.0.5",
+    "date-fns": "^2.22.1",
```

Comment:
No hats in dependencies please, this needs to be strict

Reply:
hats off!